docs/reference/images.md

```diff
@@ -336,7 +337,7 @@ $ sudo ig image inspect -o custom --extra-info=ebpf.sections ghcr.io/inspektor-g
 ["",".strtab",".text","kprobe/tcp_v4_connect",".relkprobe/tcp_v4_connect","kretprobe/tcp_v4_connect",".relkretprobe/tcp_v4_connect","kprobe/tcp_v6_connect",".relkprobe/tcp_v6_connect","kretprobe/tcp_v6_connect",".relkretprobe/tcp_v6_connect","kprobe/tcp_close",".relkprobe/tcp_close","kprobe/tcp_set_state",".relkprobe/tcp_set_state","kretprobe/inet_csk_accept",".relkretprobe/inet_csk_accept",".rodata",".bss","license",".maps",".BTF",".rel.BTF",".BTF.ext",".rel.BTF.ext",".llvm_addrsig",".symtab"]
 
 # Listing programs
-$ sudo ig image inspect -o custom fsnotify --extra-info ebpf.programs|jq -r '.[].Section'
+$ sudo ig image inspect -o custom fsnotify --extra-info ebpf.programs --jsonpath='[*].Section'
 WARN[0001] image signature verification is disabled due to using corresponding option 
 kprobe/fanotify_handle_event
```

Comment:
- `-o custom` should be removed
- The output in the documentation should be updated.

Ditto in other commands